satpy/enhancements/__init__.py

```diff
@@ -140,6 +140,75 @@ def func(band_data):
     return apply_enhancement(img.data, func)
 
 
+def reinhard(img, saturation=1.25, luminosity=1, **kwargs):
+    """Stretch method based on the Reinhard algorithm.
+
+    Luminosity has default value 1. Less is darker.
+    Saturation has default value 1.25. Less is grayer.
```

Comment:
Maybe convert this to a regular `Args:` section with this information included?

Reply:
Fixed